WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -559,10 +564,11 @@ def __doOSG ( self ):
     """ Treat the OSG case """
 
     osgDir = ''
+    vo = self.pp.releaseProject.replace( 'DIRAC', '' ).upper()
```

Comment:
This function essentially creates extra directory for the pilot, apparently there was a good reason for that although this looks strange as this should be handled by the batch system. I do not see a good reason for defining VO_%s_SW_DIR variable as there are no uses of it unless there will be a pilot command for checking the app software. As for the VO, I think pilot can get an option /DIRAC/VirtualOrganization since it is not mixing different VOs. However, I do not know if vo.upper() to be used as it is done on doOSG() will be correct for all the VOs. Please,decide what do you actually need for the OSG case based on your experience.